src/buildercore/core.py

```diff
@@ -352,13 +350,17 @@ def single_node_work():
                     continue
                 else:
                     raise err
-            except config.FabricException as err:
+            except command.FabricException as err:
                 LOG.error(str(err).replace("\n", "    "))
                 # available as 'results' to fabric.tasks.error
                 raise err
 
     # something less stateful like a context manager?
-    output['aborts'] = False
+    # lsh@2019-10: these params are passed to the settings context manager but are
+    # still (deliberately) quite stateful.
```

Comment:
update comment. behaviour of these params is preserved despite being passed to a context manager